docs/content/docs/data-structures/sequence.md

```diff
@@ -0,0 +1,198 @@
+    sharedString.annotateRange(
+        0,
+        sharedString.getLength(),
+        { decoration: "underline" });
+    props1 = sharedString.getPropertiesAtPosition(1);
+    props5 = sharedString.getPropertiesAtPosition(5);
+    // props1 = { weight: 5, decoration: "underline" }
+    // props5 = { decoration: "underline" }
+
+    // remove property called weight on all positions
+    sharedString.annotateRange(
+        0,
+        sharedString.getLength(),
+        { weight: null });
+    props1 = sharedString.getPropertiesAtPosition(1);
+    props5 = sharedString.getPropertiesAtPosition(5);
+    // props1 = { decoration: "underline" }
+    // props5 = { decoration: "underline" }
+```
+
+Whenever an operation is performed on a sequence a *sequenceDelta* event will be raised. This even provides the ranges
+affected by the operation, the type of the operation, and the properties that were changes by the operation.
+
+## How Collaboration Works
+
+The Fluid sequence data structures are eventually consistent, which means all collaborators will end up in the same
+final state. However, the intermediate states seen by each collaborator may not be seen by other collaborators. These
+intermediate states occur when two or more collaborators modify the same position in the sequence which results in a
+conflict.
+
+The basic strategy for insert conflict resolution in the sequence is to merge *far*. This strategy depends on a
+fundamental property of the Fluid Framework, which is guaranteed ordering. So, if two or more collaborators perform an
+operation on a sequence, the operations will be given an ordering and all clients will see those operations in the same
+order. What this means for the merge *far* strategy for resolving conflicting inserts is that the first operation will
+be placed in the conflicting position when it is received. When the next insert with the same position arrives and is
+applied it will be placed at the specified position and the previous inserts content position will be increased by the
+length of the incoming content pushing is farther towards the length of the sequence. This is what we call merging
+*far*.
+
+Like insert the strategies for remove and annotate also rely on guaranteed ordering. For remove and annotate only
+content visible to the collaborator creating the operation will be modified, any content ordered after the won't be.
+
+For remove this means we can't have an insert and a remove at the same time, as they will have an order, and all
+collaborators will see the operations in the same order. We also detect overlapping removes made by different
+collaborators, the resolutions here is straightforward -- the content is removed.
+
+As mentioned above, annotate operations behave like operations on SharedMaps. The merge strategy here is last one wins.
+So, if two collaborators set the same key on the annotate properties the operation that gets ordered last will
+determine the value.
+
+## SharedString
```

Comment:
Many of the code examples above used sharedString, so isn't most of this section redundant? Maybe what we need is only a section on the ways that SharedStrings differ from other sequences. E.g., the markers stuff.